messaginginapp/src/sharedTest/java/io/customer/messaginginapp/ModuleMessagingInAppTest.kt

```diff
@@ -32,17 +37,16 @@ internal class ModuleMessagingInAppTest : BaseTest() {
         module = ModuleMessagingInApp(
             moduleConfig = MessagingInAppModuleConfig.Builder().setEventListener(eventListenerMock)
                 .build(),
-            overrideDiGraph = di,
-            organizationId = "test"
+            overrideDiGraph = di
         )
     }
 
     @Test
-    fun initialize_givenComponentInitialize_expectGistToInitializeWithCorrectOrganizationId_expectModuleHookToBeAdded_expectSubscriptionOfGistCallbacks() {
+    fun initialize_givenComponentInitialize_expectGistToInitializeWithCorrectSiteIdAndRegion_expectModuleHookToBeAdded_expectSubscriptionOfGistCallbacks() {
```

Comment:
May be we should rename it to simpler version and add comments/docs if needed? 🤔 
```suggestion
    fun initialize_givenComponentInitialize_expectGistToInitializeWithCorrectValuesAndHooks() {
```